codegen/projections/rails_json/lib/rails_json/middleware.rb

```diff
@@ -53,8 +53,10 @@ def self.build(config, options = {})
         stack.use(Middleware::RequestId)
         stack.use(Hearth::Middleware::Send,
           client: config.http_client,
+          response_events: false,
           stub_data_class: Stubs::AllQueryStringTypes,
           stub_error_classes: [],
+          stub_message_encoder: Hearth::EventStream::Binary.const_get(:MessageEncoder).new,
```

Comment:
Why do you need const_get here?

Reply:
This is how the module is intended to work from codegen - similar to the way we do an error module.

The specific encoding of messages is protocol specific (all existing ones currently use this same logic, but its supposed to be configurable in codegen per protocol) - so the protocol itself defines just a module to use and that module is expected to porovide a `MessageEncoder` and a `MessageDecoder`.